### PR TITLE
feat(docs): add gulp task to generate the svg asset cache

### PR DESCRIPTION
**Pull request by DevVersion**
- Added Gulp Task for Asset Cache
- Updated `CODPEN.MD` (also for #6727)
- Removed old Asset Cache Build Script
- Updated Asset Cache from the Gulp Task

@ThomasBurleson Can you take a look? :)

### CODE REVIEW

File: docs/guides/CODEPEN.md [hunk 1]
```diff
@@ -67,7 +64,7 @@ CDN is located on the Codepen PRO account.
 
 ## Deployment Considerations
 
-The step to generate and deploy the asset-cache.js is currently a
+The step to generate and deploy the svg-assets-cache.js is currently a
```
Comment (ThomasBurleson) [inline docs/guides/CODEPEN.md]: only deployment to the CDN (and copying of the URL) is manual.

Comment (DevVersion) [inline docs/guides/CODEPEN.md]: @ThomasBurleson should be good now ;)

**Comment by ThomasBurleson** [comment]
@DevVersion - I am not inclined to accept this PR. 

*  the file is not a template; so `docs/config/template/svg-assets-cache.template.js` is not good.
*  this gulp task is fragile:
  * Often the definitions in the standalone SVG file need to be modified to be used in a icon set. For example, see `docs/app/img/icons/ic_euro_24px.svg` and see its definition entry in the `svg-assets-cache.js`

**Comment by DevVersion** [comment]
@ThomasBurleson 
Wow, I'm surprised, I missed a few things.



To point 2, I can't detect any schema in the given SVG, if there are always other things to modify, we need to detect these elements. Is there anything I need to know about modifying the SVG's?

About the `javascript` template, to be honest, I don't know what template engine is preferred in this case?

**Comment by ThomasBurleson** [comment]
Closing this PR as too fragile/brittle and not related to our core efforts.

**Pull request closed without merging**